Makefile

```diff
@@ -148,6 +148,16 @@ distclean:
 install: checkprefix nvim
 	+$(BUILD_TOOL) -C build install
 
+clang-tidy: generate-compilation-db
```

Comment:
Running `make clang-tidy` gives 

```
Error: no input files specified.
USAGE: clang-tidy [options] <source0> [... <sourceN>]
```

Reply:
sorry, this must've broke when I removed the shell script runner. It should be good now.